app/policies/article_policy.rb

```diff
@@ -144,13 +144,44 @@ def subscriptions?
     user_author? || user_super_admin?
   end
 
+  def elevated_user?
+    user_any_admin? || user_moderator?
+  end
+
   # this method performs the same checks that determine
   # if the record can be featured
   def revoke_publication?
     require_user!
     return false unless @record.published?
 
-    user_any_admin? || user_moderator?
+    elevated_user?
+  end
+
+  def allow_tag_adjustment?
```

Comment:
I'm feeling a very "The names of these methods are only going to reveal themselves later." vibe.

In other words, what's the difference between the `adjust_all_tags?` and `allow_tag_adjustment?`;  I think I'm stumbling on the `all` part.